model/annotations/task_annotations.go

```diff
@@ -210,3 +212,24 @@ func UpdateAnnotation(a *TaskAnnotation, userDisplayName string) error {
 	)
 	return errors.Wrapf(err, "problem adding task annotation for '%s'", a.TaskId)
 }
+
+// ValidateIssueURL checks the URL of an IssueLink to make sure that it has a scheme and a host.
+func ValidateIssueURL(issueURL string) error {
```

Comment:
This function seems to duplicate the intent of [IsURL](https://github.com/evergreen-ci/evergreen/blob/822f3633bfa1e9f8c95d49f91c602e0b65bb7072/graphql/util.go#L112). Maybe we should consolidate them?

Reply:
I didn't know that function existed. Thank you for pointing that out! Should the two functions be consolidated into one despite `ValidateIssueURL` being more for helping the user spot specific errors in the URLs of their IssueLinks?

Reply:
what is there about ValidateIssueURL that's specific to issues and wouldn't apply to any URL?

Reply:
Good point! I'll try to consolidate the two